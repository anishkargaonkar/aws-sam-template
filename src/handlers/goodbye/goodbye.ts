import { APIGatewayProxyResult } from 'aws-lambda';

export const handler = async (): Promise<APIGatewayProxyResult> => {
	
  return { body: JSON.stringify({ message: 'goodbye' }), statusCode: 200 };
};
