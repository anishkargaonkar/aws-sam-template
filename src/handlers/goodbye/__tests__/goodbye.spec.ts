import { handler } from "../goodbye";

describe('hello handler', () => {
  it('should say hello', async () => {
    const result = await handler();
    expect(result).toMatchSnapshot();
  });
});
