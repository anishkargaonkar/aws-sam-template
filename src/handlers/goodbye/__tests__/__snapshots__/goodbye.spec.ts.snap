// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello handler should say hello 1`] = `
Object {
  "body": "{\\"message\\":\\"goodbye\\"}",
  "statusCode": 200,
}
`;
